templates/webapi/branding/openSUSE/external_reporting.html.ep

```diff
@@ -76,7 +76,10 @@
 %     }
 % }
 % elsif ($raw_distri eq 'sle-micro') {
+%     # "Enterprise" is removed in 6.1 and 6.2
+%     my $version = $job->VERSION =~ s/-/ /r;
```

Comment:
not sure I get this. is VERSION usually x.y? Why the hypher subtitute?

Reply:
Yes, version in SL(E)-Micro is always `x.y`. Maybe not needed, I copy-pasted from another example. Let me remove it